board/safety/safety_tesla.h

```diff
@@ -0,0 +1,319 @@
+// board enforces
+//   in-state
+//      accel set/resume
+//   out-state
+//      cancel button
+//      regen paddle
+//      accel rising edge
+//      brake rising edge
+//      brake > 0mph
+
+// 2m/s are added to be less restrictive
+const struct lookup_t TESLA_LOOKUP_ANGLE_RATE_UP = {
+    {2., 7., 17.},
+    {5., .8, .25}};
```

Comment:
above 17mph, this is twice than what is in `toyota_safety_ipas.h`. Isn't it too much?

Reply:
I'm sorry for my ignorance, I didn't copy this piece of code.  It looks like the difference is the .25 in Tesla, which is .15 in Toyota.  Is that the part that concerns you?

Reply:
yes

Reply:
So we have a couple engineers and coders who tested and tweaked the values in this code and I believe that that value was necessary in order to take even some modest curves at a decent speed.  I don't think it's excessive.  There are still some curves that the model can't take which I believe it should.  I can definitely change this to .15 for upstreaming purposes, but I have a feeling users will likely change it back because of the turns it would disengage on.  Definitely your call, please let me know.

Reply:
ok, let's keep these numerical details last. It comes down to testing the max steer injection case:
https://medium.com/@comma_ai/how-to-write-a-car-port-for-openpilot-7ce0785eda84